### PR TITLE
[NTH-85] User Login Form Submission and Access Token Storage

### PR DESCRIPTION
**Pull request by raghdarizq**
- When the button is clicked, a request is sent to the server's '/userLogin' endpoint to authenticate the user.
- The email and password values entered in the form are included in the request's payload as data.
- If the server responds with a successful authentication, the response object, which likely includes an access token, is stored in the browser's local storage.
- The access token is stored under the key 'access_token'

### CODE REVIEW

File: src/pages/login/index.tsx [hunk 1]
```diff
@@ -22,17 +24,29 @@ import {
   gridStyle,
 } from './classes';
 import './style.css';
+import axiosInstance from '../../utils/apis/axios';
```
Comment (ahmedmurtaja) [inline src/pages/login/index.tsx]: import from api index

File: src/pages/login/index.tsx [hunk 2]
```diff
@@ -22,17 +24,29 @@ import {
   gridStyle,
 } from './classes';
 import './style.css';
+import axiosInstance from '../../utils/apis/axios';
 
 const Login = () => {
+  const { enqueueSnackbar } = useSnackbar();
```
Comment (ahmedmurtaja) [inline src/pages/login/index.tsx]: you can import it directly from 'notistack';

**Review by ahmedmurtaja — CHANGES_REQUESTED** [review]

File: src/pages/login/index.tsx [hunk 3]
```diff
@@ -22,22 +24,36 @@ import {
...
+    onSubmit: async (values) => {
+      try {
+        const response = await axiosInstance.post('/auth/login', {
+          email: values.email,
+          password: values.password,
+        });
+        localStorage.setItem('access_token', response.data.access_token);
+      } catch (error) {
+        showSnackbar(error.message, 'error');
+      }
```
Comment (LinaYahya) [inline src/pages/login/index.tsx]:
```suggestion
 } catch (error) {
        if (error instanceof Error) {
          showSnackbar(error.message, 'error');
        } else {
                  showSnackbar('something went wrong', 'error');

      }

```

File: src/pages/login/index.tsx [hunk 4]
```diff
@@ -22,22 +24,36 @@ import {
...
-    onSubmit: (values) => {
-      // eslint-disable-next-line no-console
-      console.log(JSON.stringify(values, null, 2));
+    onSubmit: async (values) => {
+      try {
+        const response = await axiosInstance.post('/auth/login', {
+          email: values.email,
+          password: values.password,
+        });
+        localStorage.setItem('access_token', response.data.access_token);
```
Comment (LinaYahya) [inline src/pages/login/index.tsx:50]: let us redirect to home when login is success

**Review by LinaYahya — CHANGES_REQUESTED** [review]

**Review by LinaYahya — APPROVED** [review]

**Review by MohannadSabaa — APPROVED** [review]

**Review by LinaYahya — APPROVED** [review]

**Review by ahmedmurtaja — APPROVED** [review]

**Review by YasserAmro0 — APPROVED** [review]

**Pull request merged**